From 984d747cfff5b6b21d64bf02c1ced8f6eb558f83 Mon Sep 17 00:00:00 2001
From: Christian Hofstaedtler <christian@hofstaedtler.name>
Date: Sun, 1 Jun 2014 17:24:19 +0200
Subject: [PATCH] Support building recursor on kFreeBSD and Hurd

---
 Makefile | 6 ++++++
 kqueuemplexer.cc  | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 300ff71..d00a817 100644
--- a/Makefile
+++ b/Makefile
@@ -33,6 +33,12 @@ all: message version_generated.h build
 
 # OS specific instructions
 -include sysdeps/$(shell uname).inc
+ifeq ($(shell uname),GNU/kFreeBSD)
+	-include sysdeps/FreeBSD.inc
+endif
+ifeq ($(shell uname),GNU/Hurd)
+	-include sysdeps/Linux.inc
+endif
 
 ifeq ($(LUA), 1)
 	LUALIBS=$(LUA_LIBS_CONFIG)
diff --git a/kqueuemplexer.cc b/kqueuemplexer.cc
index 6f89223..c078457 100644
--- a/kqueuemplexer.cc
+++ b/kqueuemplexer.cc
@@ -6,7 +6,7 @@
 #include <boost/lexical_cast.hpp>
 #include "syncres.hh"
 #include <sys/types.h>
-#ifdef __FreeBSD__ 
+#if defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
 #include <sys/event.h>
 #endif
 #include <sys/time.h>
-- 
1.9.3

